scss/_tables.scss

```diff
@@ -129,6 +129,21 @@
       border-color: $table-border-color;
     }
   }
+
+  // Sticky header
+  //
+  // Table header with the `.thead-sticky` class applied sticks its cells on top
+  // allowing only <tbody> to scroll.
+
+  @supports (position: sticky) {
+    .thead-sticky {
+      th {
+        position: sticky;
+        top: $table-head-sticky-top;
+        z-index: $zindex-sticky;
```

Comment:
Checking this out on the built Netlify preview, I'm wondering if this is the right `z-index` level. https://deploy-preview-29456--twbs-bootstrap.netlify.com/docs/4.3/content/tables/#sticky-header

This stuff shouldn't overlay any other fixed, stickied, absolute, etc navbars and other items. I feel like this just needs to overlay the content that comes after it. Thoughts @twbs/css-review?

Reply:
I think we better set a custom offset in our docs, so that the sticky header is displayed below the sticky header. If we just decrease the `z-index`, the documentation won't be clear because we won't see the sticky table head.

And we should of course document the `$table-head-sticky-top` too.

Reply:
> I think we better set a custom offset in our docs, so that the sticky header is displayed below the sticky header. If we just decrease the `z-index`, the documentation won't be clear because we won't see the sticky table head.
Make totally sense. Using `$zindex-sticky` for more than one sticky component could and should end up with conflict. I'm not feeling comfortable to suggest another var name.

Reply:
Left you hanging here @ed1nh0. I'd suggest we do `$table-head-sticky-zindex` if you're still down to iterate here.

Reply:
> Left you hanging here @ed1nh0. I'd suggest we do `$table-head-sticky-zindex` if you're still down to iterate here.

Yeah sure! But as for syntax standard I'd suggest swap its end, therefore `$table-head-zindex-sticky`, what do you think?

Reply:
What if the table is wrapped in a vertically scrollable div...  Where the sticky headers should appear under any sticky top navbars when the page is scrolled?

I would expect this (table in a scrollable div) to be the more common use case than having the table header stick to the top of the page.

Reply:
We found that a z-index of 2 to 5 works well.